test/articledownloadlinkslist.spec.js

```diff
@@ -11,10 +17,39 @@ describe('An ArticleDownloadLinksList Component', function () {
 
   beforeEach(function () {
     articleDownloadLinksList = new ArticleDownloadLinksList($el, window, window.document);
+    spy(articleDownloadLinksList.$elm, 'setAttribute');
+  });
+
+  afterEach(function() {
+    articleDownloadLinksList.$elm.setAttribute.restore();
+  });
+
+  describe('The open method', function () {
+
+    it('sets aria-expanded attribute to "true"', function () {
+      articleDownloadLinksList.open();
+      expect(articleDownloadLinksList.$elm.setAttribute.calledWithExactly('aria-expanded', "true")).to.be.true;
+    })
+
+    it('removes the class "visuallyhidden"', function () {
+      articleDownloadLinksList.open();
+      expect(articleDownloadLinksList.$elm.classList.contains('visuallyhidden')).to.be.false;
+    });
+
   });
 
-  it('exists', function () {
-    expect(articleDownloadLinksList).to.exist;
+  describe('The close method', function () {
+
+    it('sets aria-expanded attribute to "false"', function () {
+      articleDownloadLinksList.close();
+      expect(articleDownloadLinksList.$elm.setAttribute.calledWithExactly('aria-expanded', "false")).to.be.true;
+    });
+
+    it('adds the class "visuallyhidden"', function () {
+      articleDownloadLinksList.close();
+      expect(articleDownloadLinksList.$elm.classList.contains('visuallyhidden')).to.be.true;
+    });
+
   });
 
-});
+});
```

Comment:
Convention is that all files should finish with an empty line, so just add a return here.

Reply:
Completed